staging/src/k8s.io/api/flowcontrol/v1beta1/types.go

```diff
@@ -57,6 +57,11 @@ const (
 	ResponseHeaderMatchedFlowSchemaUID                 = "X-Kubernetes-PF-FlowSchema-UID"
 )
 
+const (
+	// AutoUpdateAnnotationKey is the name of an annotation which prevents reconciliation if set to "false"
+	AutoUpdateAnnotationKey = "apf.kubernetes.io/autoupdate"
```

Comment:
i think we can use a better name, how about `configuration.apf.kubernetes.io/autoupdate`?

Reply:
How about `apf.kubernetes.io/autoupdate-spec` ?